manifest.json

```diff
@@ -11,12 +11,9 @@
   "dependencies": {
     "vtex.styleguide": "9.x"
   },
-  "peerDependencies": {
-    "vtex.admin": "2.x"
-  },
   "builders": {
     "react": "3.x",
-    "pages": "1.x",
+    "admin": "0.x",
```

Comment:
Shouldn't it be `2.x`?

Reply:
nope, it's the builder, not the admin app 😛